android/libraries/rib-base/src/main/java/com/uber/rib/core/InteractorComponent.java

```diff
@@ -21,9 +21,8 @@
  * Presidio Intellij Plugin.
  *
  * @param <T> type of interactor that is injected.
- * @param <P> type of presenter.
  */
-public interface InteractorComponent<P extends Presenter, T extends Interactor<P, ?>>
+public interface InteractorComponent<T extends Interactor<?>>
```

Comment:
I believe we don't need it
You can provide an interactor directly into router constructor